src/bundle.c

```diff
@@ -612,7 +612,7 @@ int add_subscriptions(struct list *bundles, struct list **subs, int current_vers
 
 		file = search_bundle_in_manifest(mom, bundle);
 		if (!file) {
-			fprintf(stderr, "%s bundle name is invalid, skipping it...\n", bundle);
+			fprintf(stderr, "Warning: Bundle \"%s\" is invalid, skipping it...\n", bundle);
```

Comment:
We should also make how we print `Warning` messages consistent with a `print_warn` function or something. I'll open #489 to track.

Reply:
I like that idea